base/strings/search.jl

```diff
@@ -101,8 +101,24 @@ julia> findfirst("Julia", "JuliaLang")
 1:5
 ```
 """
-findfirst(pattern::AbstractString, string::AbstractString) =
-    findnext(pattern, string, firstindex(string))
+findfirst(pattern::AbstractString, string::AbstractString) = findnext(pattern, string, firstindex(string))
+
+"""
+    findfirst(ch::Char, string::AbstractString)
+
+Find the first occurrence of `ch` in `string`. Equivalent to
+[`findfirst(isequal(ch), string)`](@ref).
+
```

Comment:
As @fredrikekre noted, please add this to both docstrings:
```julia
!!! compat "Julia 1.3"
    This function requires at least Julia 1.3.
```

Please also add an entry to NEWS.md.
```